node/metrics/metrics.go

```diff
@@ -47,6 +47,8 @@ func InitBaseMetrics(appStatusHandler core.AppStatusHandler) error {
 	appStatusHandler.SetUInt64Value(common.MetricAccountsSnapshotNumNodes, initUint)
 	appStatusHandler.SetUInt64Value(common.MetricTrieSyncNumProcessedNodes, initUint)
 	appStatusHandler.SetUInt64Value(common.MetricTrieSyncNumReceivedBytes, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricAccountsSnapshotInProgress, initUint)
```

Comment:
Why do we require this change?

Reply:
it was requested by Bogdan and Liviu, to be easier for parsing, not sure if it's a hard requirement though

Reply:
because it's easier in our monitoring tools to parse these values as int (0-false, 1-true) instead of booleans